packages/shared/src/utils/helpers/duration.ts

```diff
@@ -52,8 +52,9 @@ export const buildDurationConfig = (
     const duration_min_max = durations.min_max[contract.start_type as keyof typeof durations.min_max];
     const obj_min = getDurationFromString(contract.min_contract_duration);
     const obj_max = getDurationFromString(contract.max_contract_duration);
+    const new_durations = Object.create(durations);
 
-    durations.min_max[contract.start_type as keyof typeof durations.min_max][
+    new_durations.min_max[contract.start_type as keyof typeof durations.min_max][
```

Comment:
can we use this `new_durations` for other `durations` in this file?

Reply:
@ali-hosseini-deriv we still have problem for this
I should find another way for fixing this